LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
+                           Version 2.0, January 2004
+                        http://www.apache.org/licenses/
+
+   TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
```

Comment:
Thanks for adding this! We still have to make add licensing details to build.gradle file since we're publishing to maven central.

Reply:
Added to build.gradle